kse/src/main/java/org/kse/crypto/secretkey/PasswordType.java

```diff
@@ -0,0 +1,178 @@
+    PBEWithHmacSHA256AndAES_256("PBEWithHmacSHA256AndAES_256", "PBE with HMac-SHA-256 and AES-256"),
+    PBEWithHmacSHA384AndAES_128("PBEWithHmacSHA384AndAES_128", "PBE with HMac-SHA-384 and AES-128"),
+    PBEWithHmacSHA384AndAES_256("PBEWithHmacSHA384AndAES_256", "PBE with HMac-SHA-384 and AES-256"),
+    PBEWithHmacSHA512AndAES_128("PBEWithHmacSHA512AndAES_128", "PBE with HMac-SHA-512 and AES-128"),
+    PBEWithHmacSHA512AndAES_256("PBEWithHmacSHA512AndAES_256", "PBE with HMac-SHA-512 and AES-256"),
+    PBEWITHHMACTIGER("PBEWITHHMACTIGER", "PBE with HMac-TIGER"),
+    PBEWITHMD2ANDDES("PBEWITHMD2ANDDES", "PBE with MD2 and DES"),
+    PBEWITHMD2ANDRC2("PBEWITHMD2ANDRC2", "PBE with MD2 and RC2"),
+    PBEWITHMD5AND128BITAES_CBC_OPENSSL("PBEWITHMD5AND128BITAES-CBC-OPENSSL", "PBE with MD5 and AES-128 CBC OpenSSL"),
+    PBEWITHMD5AND192BITAES_CBC_OPENSSL("PBEWITHMD5AND192BITAES-CBC-OPENSSL", "PBE with MD5 and AES-192 CBC OpenSSL"),
+    PBEWITHMD5AND256BITAES_CBC_OPENSSL("PBEWITHMD5AND256BITAES-CBC-OPENSSL", "PBE with MD5 and AES-256 CBC OpenSSL"),
+    PBEWithMD5AndDES("PBEWithMD5AndDES", "PBE with MD5 and DES"),
+    PBEWithMD5AndTripleDES("PBEWithMD5AndTripleDES", "PBE with MD5 and DESede"),
+    PBEWITHMD5ANDRC2("PBEWITHMD5ANDRC2", "PBE with MD5 and RC2"),
+    PBEWITHSHAAND2_KEYTRIPLEDES_CBC("PBEWITHSHAAND2-KEYTRIPLEDES-CBC", "PBE with SHA-1 and 2-key TripleDES CBC"),
+    PBEWITHSHAAND3_KEYTRIPLEDES_CBC("PBEWITHSHAAND3-KEYTRIPLEDES-CBC", "PBE with SHA-1 and 3-key TripleDES CBC"),
+    PBEWITHSHAAND128BITAES_CBC_BC("PBEWITHSHAAND128BITAES-CBC-BC", "PBE with SHA-1 and AES-128 CBC BC"),
+    PBEWITHSHAAND192BITAES_CBC_BC("PBEWITHSHAAND192BITAES-CBC-BC", "PBE with SHA-1 and AES-192 CBC BC"),
+    PBEWITHSHAAND256BITAES_CBC_BC("PBEWITHSHAAND256BITAES-CBC-BC", "PBE with SHA-1 and AES-256 CBC BC"),
+    PBEWITHSHA1ANDDES("PBEWITHSHA1ANDDES", "PBE with SHA-1 and DES"),
+    PBEWithSHA1AndDESede("PBEWithSHA1AndDESede", "PBE with SHA-1 and DESede"),
+    PBEWITHSHAANDIDEA_CBC("PBEWITHSHAANDIDEA-CBC", "PBE with SHA-1 and IDEA CBC"),
+    PBEWITHSHA1ANDRC2("PBEWITHSHA1ANDRC2", "PBE with SHA-1 and RC2"),
+    PBEWithSHA1AndRC2_128("PBEWithSHA1AndRC2_128", "PBE with SHA-1 and RC2-128"),
+    PBEWithSHA1AndRC2_40("PBEWithSHA1AndRC2_40", "PBE with SHA-1 and RC2-40"),
+    PBEWithSHA1AndRC4_128("PBEWithSHA1AndRC4_128", "PBE with SHA-1 and RC4-128"),
+    PBEWITHSHAAND128BITRC4("PBEWITHSHAAND128BITRC4", "PBE with SHA-1 and RC4-128 CBC"),
+    PBEWithSHA1AndRC4_40("PBEWithSHA1AndRC4_40", "PBE with SHA-1 and RC4-40"),
+    PBEWITHSHAANDTWOFISH_CBC("PBEWITHSHAANDTWOFISH-CBC", "PBE with SHA-1 and Twofish CBC"),
+    PBEWITHSHA256AND128BITAES_CBC_BC("PBEWITHSHA256AND128BITAES-CBC-BC", "PBE with SHA-256 and AES-128 CBC BC"),
+    PBEWITHSHA256AND192BITAES_CBC_BC("PBEWITHSHA256AND192BITAES-CBC-BC", "PBE with SHA-256 and AES-192 CBC BC"),
+    PBEWITHSHA256AND256BITAES_CBC_BC("PBEWITHSHA256AND256BITAES-CBC-BC", "PBE with SHA-256 and AES-256 CBC BC");
```

Comment:
Hello @jonwltn 

_Here is a minor review..._

For localisation reason 🌐:
- Would it be possible to avoid hardcoding all the values ​​for the EN language?

And use resource as:
https://github.com/kaikramer/keystore-explorer/blob/77e76963a3c51aa0ce50dd3ab36c486605e6be76/kse/src/main/resources/org/kse/crypto/privatekey/resources.properties#L44-L60

_Thanks a lot,_
Regards,
Th.